### PR TITLE
Refine comments in openhabian.conf

### PR DESCRIPTION
**Pull request by mstormi**
Fixes: #1143

Signed-off-by: Markus Storm <markus.storm@gmx.net>

### CODE REVIEW

**Review by ecdye — CHANGES_REQUESTED** [review]

File: build-image/openhabian.conf [hunk 1]
```diff
@@ -1,19 +1,24 @@
...
+# Hostname to set this one to
+hostname=openHABianDevice
+# the default user (with its working environment) will be renamed to this username given here ...
 username=openhabian
-userpw=openhabian
+# ... and given this password. The password in turn will be removed from this file after completion
+userpw=password
 
-# Language and time. Be sure to set these to valid values
-timeserver=0.pool.ntp.org
```
[the remark this answers is not in the document]
Comment (mstormi) [reply build-image/openhabian.conf:9]: I don't know. I didn't find it to be used anywhere in the code hence decided to remove it.

File: build-image/openhabian.conf [hunk 2]
```diff
@@ -1,19 +1,24 @@
...
+# If you want to change anything after unattended install completed, use menu in openhabian-config.
 
-hostname=openhab
+# Hostname to set this one to
+hostname=openHABianDevice
+# the default user (with its working environment) will be renamed to this username given here ...
 username=openhabian
-userpw=openhabian
+# ... and given this password. The password in turn will be removed from this file after completion
+userpw=password
```
[the remark this answers is not in the document]
Comment (mstormi) [reply build-image/openhabian.conf:11]: err right, that wasn't my intention. We want the user to change the password this way.
Changed that back.

File: docs/openhabian.md [hunk 3]
```diff
@@ -120,7 +120,17 @@ ln -s /opt/openhabian/openhabian-setup.sh /usr/local/bin/openhabian-config
 cp /opt/openhabian/openhabian.conf.dist /etc/openhabian.conf
 ```
 
-Edit `/etc/openhabian.conf` to match your needs, then use
+#### interactive install on generic Linux
```
[the remark this answers is not in the document]
Comment (mstormi) [reply docs/openhabian.md]: ok

File: build-image/openhabian.pi-raspios64beta.conf [hunk 4]
```diff
@@ -1,19 +1,24 @@
...
+# If you want to change anything after unattended install completed, use menu in openhabian-config.
 
-hostname=openhab
+# Hostname to set this one to
+hostname=openHABianDevice
+# the default user (with its working environment) will be renamed to this username given here ...
 username=openhabian
-userpw=openhabian
+# ... and given this password. The password in turn will be removed from this file after completion
+userpw=password
```
[the remark this answers is not in the document]
Comment (mstormi) [reply build-image/openhabian.pi-raspios64beta.conf]: changed too

**Review by ecdye — APPROVED** [review]
LGTM

**Pull request merged**